packages/components/src/Progress/Progress.scss

```diff
@@ -40,7 +40,7 @@ $gradient-part-width: 100% / 6;
 		&.infinite {
 			background: transparent;
 			width: 100%;
-			animation: infinite-progression-keyframes 5s infinite;
+			animation: infinite-progression-keyframes 2s infinite;
```

Comment:
The duration is not defined on linked frontify specification, can you ask UX/UI team to update spec